spec.bs

```diff
@@ -2209,12 +2214,16 @@ Once that web driver changes is made, existing Chromium-internal web platform te
 </div>
 
 <div algorithm>
-  To determine whether fetching a [=request=] <var ignore>request</var> <dfn>must be blocked due to
-  a revoked partition nonce</dfn> using a [=fenced frame config instance/partition nonce=] |nonce|
-  and a [=URL=] |requestURL|, run these steps:
+  To determine whether fetching a [=request=] |request| <dfn>must be blocked due to
+  a revoked partition nonce</dfn>, run these steps:
```

Comment:
I'm guessing this can be unwrapped more optimally on the previous line now? Try running `specfmt`, on the latest version.